src/components/HelpDrawer/HelpContent/StatsHelpContent.tsx

```diff
@@ -1,19 +1,25 @@
 import {ImageComponent} from "../ImageComponent";
-import headStatisticsButton from "static/help/head_statistics_button.png";
-import headStatisticsButton_d from "static/help/head_statistics_button_d.png";
+import widgetButtonStatistics from "static/help/widgetButton_statistics.png";
+import widgetButtonStatistics_d from "static/help/widgetButton_statistics_d.png";
 
 export const STATS_HELP_CONTENT = (
     <div>
         <p>
-            <ImageComponent light={headStatisticsButton} dark={headStatisticsButton_d} width="90%" />
+            <ImageComponent light={widgetButtonStatistics} dark={widgetButtonStatistics_d} width="90%" />
         </p>
         <p>Statistics widget allows you to view statistical quantities over a 2D region. When no region is created or selected, it displays statistical quantities of the full image in the image viewer.</p>
-        <h3 id="images">Images</h3>
-        <p>The image dropdown defaults to &quot;Active&quot; image which means the current image in the image viewer.</p>
-        <h3 id="regions">Regions</h3>
+        <h3 id="images">Image</h3>
         <p>
-            The region dropdown defaults to &quot;Active&quot; region which means a selected region in the image viewer. You can select a region by clicking one on the image viewer, or by clicking a region entry on the region list widget.
-            Statistics of the selected region will be updated accordingly.
+            The <code>Image</code> dropdown menu defaults to &quot;Active&quot; image which means the current image in the image viewer if it is single-panel mode. If it is multi-panel mode, the active image is highlighted with a red box.
+        </p>
+        <h3 id="regions">Region</h3>
+        <p>
+            The <code>Region</code> dropdown menu defaults to &quot;Active&quot; region which means a selected region in the image viewer. You can select a region by clicking one on the image viewer, or by clicking a region entry on the
```

Comment:
"the region selected in the image viewer"

"clicking on one in the image viewer"

"clicking on a region entry in the region list widget"